javascript/javascript1/week1/homework.js

```diff
@@ -0,0 +1,91 @@
+// Step 2
+// My freecodecamp username - @Natata08
+
+//Step 3
+// Age-ify (A future age calculator)
+
+const yearOfBirth = 1986; //number
+const yearFuture = 2025; //number
+const age = yearFuture - yearOfBirth;
+const messageAboutFututeAge = 'You will be ' + age + ' years old in ' + yearFuture;
+
+console.log(messageAboutFututeAge);
+
+// Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = false; //boolean
+let messageAboutDogAge;
+
+if (shouldShowResultInDogYears) {
+  messageAboutDogAge = 'Your dog will be ' + dogYear * 7 + ' dog years old in ' + dogYearFuture;
+} else {
+  messageAboutDogAge = 'Your dog will be ' + dogYear + ' human years old in ' + dogYearFuture;
+}
+
+console.log(messageAboutDogAge);
+
+// Housey pricey (A house price estimator)
+
+function estimateHousePrice(
+  name,
+  houseWidthInM,
+  houseHeightInM,
+  houseDepthInM,
+  gardenSizeInM2,
+  announcedHousePrice
+) {
+  const volumeInMeters = houseWidthInM * houseHeightInM * houseDepthInM;
+  const housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+  let message;
+  if (housePrice < announcedHousePrice) {
+    message = name + ' is paying too much for this house. The real price is: ' + housePrice;
```

Comment:
You could make nice use of string template literals here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Example:

```javascript
const weather = "sunny"
const degrees = 25
const outputString = `The weather is ${weather} at ${degrees} degrees.`
console.log(outputString)
```

Reply:
It's amazingly more convenient! Thanks, I've implemented it everywhere.